<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>316</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>323</width>
    <height>316</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>323</width>
    <height>316</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>307</width>
      <height>274</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Количесвот вершин</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox">
           <property name="maximumSize">
            <size>
             <width>37</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_XY">
           <property name="text">
            <string>Координаты точек</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_x1">
               <property name="maximumSize">
                <size>
                 <width>19</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>X1=</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_x2">
               <property name="maximumSize">
                <size>
                 <width>19</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>X2=</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_x3">
               <property name="maximumSize">
                <size>
                 <width>19</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>X3=</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_x4">
               <property name="maximumSize">
                <size>
                 <width>19</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>X4=</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_x5">
               <property name="maximumSize">
                <size>
                 <width>19</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>X5=</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_x6">
               <property name="maximumSize">
                <size>
                 <width>19</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>X6=</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_x7">
               <property name="maximumSize">
                <size>
                 <width>19</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>X7=</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_x8">
               <property name="maximumSize">
                <size>
                 <width>19</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>X8=</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLineEdit" name="lineEdit_x1">
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_x2">
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_x3">
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_x4">
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_x5">
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_x6">
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_x7">
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_x8">
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_y1">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Y1=</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_y2">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Y2=</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_y3">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Y3=</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_y4">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Y4=</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_y5">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Y5=</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_y6">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Y6=</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_y7">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Y7=</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_y8">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Y8=</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLineEdit" name="lineEdit_y1">
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_y2">
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_y3">
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_y4">
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_y5">
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_y6">
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_y7">
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_y8">
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Вычислить</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_area">
           <property name="maximumSize">
            <size>
             <width>21</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>S =</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_area">
           <property name="maximumSize">
            <size>
             <width>61</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_perimeter">
           <property name="maximumSize">
            <size>
             <width>21</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>P =</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_perimeter">
           <property name="maximumSize">
            <size>
             <width>61</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_save">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_saven">
         <property name="text">
          <string>Сохраненное</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>323</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
